packages/dev/loaders/src/USD/usdFileLoader.ts

```diff
@@ -0,0 +1,198 @@
+import type { ISceneLoaderPluginAsync, ISceneLoaderPluginFactory, ISceneLoaderAsyncResult, ISceneLoaderProgressEvent } from "core/Loading/sceneLoader";
+import { registerSceneLoaderPlugin } from "core/Loading/sceneLoader";
+import { USDFileLoaderMetadata } from "./usdFileLoader.metadata";
+import { AssetContainer } from "core/assetContainer";
+import type { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+import type { AbstractMesh } from "core/Meshes/abstractMesh";

```

Comment:
For new code, I would consolidate type imports for improved readability:
```suggestion
import type { ISceneLoaderPluginAsync, ISceneLoaderPluginFactory, ISceneLoaderAsyncResult, ISceneLoaderProgressEvent, Scene, Nullable, AbstractMesh } from "core/index";
import { registerSceneLoaderPlugin } from "core/Loading/sceneLoader";
import { USDFileLoaderMetadata } from "./usdFileLoader.metadata";
import { AssetContainer } from "core/assetContainer";
```

You will get an eslint warning that is safe to suppress (since type imports have no impact on the bundle).

Reply:
fixed